### DISCUSSION TITLE
Need some basic help starting up - not sure how to run correctly

### DISCUSSION THREAD

**Discussion by nmauro2017** [ReagentX/imessage-exporter#228] [Q&A] [answered]
Hello, this is a great project and I'm really excited to use it but I'm not sure exactly how to run it. I cloned the repository and was able to compile successfully but when I run any commands such as `imessage-exporter -d` I just get `zsh: command not found: imessage-exporter`. I'm sure I missed a step but I haven't seen anything in the documentation other than installing and then running the commands. Any idea what I'm missing here?

**Comment by ReagentX** [comment, marked as answer]
If you build it yourself you can run it with `cargo run --release -- <params>`.  In your example, you would run:

```
% cargo run --release -- -d
```

Using a clone of the repository doesn't install the program on your `PATH`.

**Reply by nmauro2017** [reply]
That worked thank you!
